.test_patterns.yml

```diff
@@ -222,9 +222,6 @@ tests:
     error_regex: "✕ should stop after max retry attempts"
     owners:
       - *sean
-  - regex: "p2p/src/client/p2p_client.integration.test.ts" # http://ci.aztec-labs.com/31c4c9821b1a533c
-    owners:
-      - *palla
```

Comment:
Let's leave this one in for a bit, in case the fix does not work, so we don't break everything for others